docs/en/reference/annotations-reference.rst

```diff
@@ -118,7 +118,7 @@ Optional attributes:
 -  **options**: Array of additional options:
 
    -  ``default``: The default value to set for the column if no value
-      is supplied.
+      is supplied. It's possible to use PHP constants here: ``options={"default": Domain::CONSTANT_VALUE}``
```

Comment:
It is possible to use php constants in any annotation value, so I don't really understand the point of this change...

Reply:
the key word is not "here", key word is a possibility of using constants which is not obvious from doctrine's documentation. this is just to let users know and redirect them to the full documentation